release/xgboost_tests/app_config.yaml

```diff
@@ -1,4 +1,4 @@
-base_image: "anyscale/ray:1.2.0"
+base_image: "anyscale/ray-ml:pinned-nightly-py38"
```

Comment:
you can also use :1.4.0 if you want.

Reply:
Sounds good. I think if there is no other blocker, the nightly image may include newer versions of non-Ray dependencies, which could be more suitable for nightly / release tests.